### DISCUSSION TITLE
大佬们安装完requirements.txt里面的文件之后运行python.exe .\gui.py显示No module named 'PySide6'该怎么办啊

### DISCUSSION THREAD

**Discussion by pointm** [viopsa233/AI-testers-Forum#45] [Q&A] [answered]
已经卸载重装 'PySide6'好几次了还是有这个问题
![image](https://github.com/viopsa233/AI-testers-Forum/assets/49180699/46e66ff9-29df-446f-aa0d-b2f76c327f06)

**Comment by pointm** [comment]
就那个WIKI客户端里面的那个代码

**Comment by ghost** [comment, marked as answer]
pip安装到anaconda目录，python.exe可能从其他目录运行，你的PATH环境变量有问题
